### PR TITLE
Replace iterators with for loop

### PR DESCRIPTION
**Pull request by Beacze**

### CODE REVIEW

File: Kitodo/src/main/java/de/sub/goobi/importer/ImportRussland.java [hunk 1]
```diff
@@ -186,18 +185,17 @@ private void setBandDetails(List<String> inListe) throws MetadataTypeNotAllowedE
         DocStruct ds = this.logicalTopstruct.getAllChildren().get(0);
         // logger.info(ds.getType().getName());
         /* zunächst alle Details durchlaufen und dem Band hinzufügenl */
-        for (Iterator<String> iter = inListe.iterator(); iter.hasNext();) {
-            String meinDetail = iter.next();
-            String meineDetailNr = meinDetail.substring(0, 3);
+        for (String bandDetial : inListe) {
+            String meineDetailNr = bandDetial.substring(0, 3);
```
Comment (henning-gerhardt) [inline Kitodo/src/main/java/de/sub/goobi/importer/ImportRussland.java]: Typo in `bandDetial` -> `bandDetail`

File: Kitodo/src/main/java/de/sub/goobi/importer/ImportRussland.java [hunk 2]
```diff
@@ -186,18 +185,17 @@ private void setBandDetails(List<String> inListe) throws MetadataTypeNotAllowedE
         DocStruct ds = this.logicalTopstruct.getAllChildren().get(0);
         // logger.info(ds.getType().getName());
         /* zunächst alle Details durchlaufen und dem Band hinzufügenl */
-        for (Iterator<String> iter = inListe.iterator(); iter.hasNext();) {
-            String meinDetail = iter.next();
-            String meineDetailNr = meinDetail.substring(0, 3);
+        for (String bandDetial : inListe) {
```
Comment (henning-gerhardt) [inline Kitodo/src/main/java/de/sub/goobi/importer/ImportRussland.java]: Typo in `bandDetial` -> `bandDetail`

File: Kitodo/src/main/java/de/sub/goobi/importer/ImportRussland.java [hunk 3]
```diff
@@ -211,12 +209,9 @@ private void setArticleDetails(List<String> inListe)
          * ermitteln
          */
         String zblID = "";
-        for (Iterator<String> iter = inListe.iterator(); iter.hasNext();) {
-            String meinDetail = iter.next();
-            if (meinDetail.substring(0, 3).equals("090")) {
-                // logger.info("ZBL-Identifier ist " +
-                // meinDetail.substring(4).trim());
-                zblID = meinDetail.substring(4).trim();
+        for (String articleDetial : inListe) {
```
Comment (henning-gerhardt) [inline Kitodo/src/main/java/de/sub/goobi/importer/ImportRussland.java]: Typo in `articleDetial` -> `articleDetail`

**Comment by henning-gerhardt** [comment]
This PR needs a rebase to fix merge conflicts.

**Comment by Beacze** [comment]
Rebased.

**Pull request merged**